tests/integration/test-webpaste-providers.el

```diff
@@ -84,5 +84,14 @@
    (expect (spy-calls-count 'webpaste--return-url) :to-equal 1)
    (expect (spy-calls-count 'webpaste--paste-text) :to-equal 0)))
```

Comment:
I haven't checked closely, but I'm fairly certain that you have to remove the last `)` on this line, otherwise the following `it`-block will be outside of the `describe`-block and have a `)` to much.

Reply:
you're right, fixed